steal-qunit.js

```diff
@@ -111,8 +111,8 @@ define([
 		if (window.Testee && window.Testee.init) {
 			Testee.init();
 		}
-		
-		if(typeof QUnit.start === "function") {
+
+		if(QUnit.assert && QUnit.assert.ok && QUnit.assert.async) {
```

Comment:
This check is truthy in 1.x as well though.  Why not just check the version directly?

```js
var qunitVersion = +QUnit.version.split('.')[0];
if(qunitVersion >= 2) {
  QUnit.start();
} else {
  QUnit.load();
}
```